markup/pandoc/convert.go

```diff
@@ -77,6 +85,51 @@ func getPandocBinaryName() string {
 	return ""
 }
 
+var versionOnce sync.Once
+
+// getPandocVersion parses the pandoc version output
+func getPandocVersion(cfg converter.ProviderConfig) (string, error) {
+	var version string
```

Comment:
You need to pull the `version` out of the function; this will fail he second time `getPandocVersion` gets called.

Reply:
Done, thanks.

Reply:
I also added a unit test which calls it twice to detect this issue.